src/index.js

```diff
@@ -0,0 +1,52 @@
+// import _ from 'lodash';
+import './style.css';
+
+// Selection of Required Elements
+const todoList = document.querySelector('#display-tasks');
+
+// Create Array to Populate DOM
+const taskList = [{
+  didComplete: 'false',
+  task: 'Clean My Room',
+  number: 0,
```

Comment:
[OPTIONAL] I think you naming 'didComplete' as 'completed', 'task' as 'description', and 'number' as 'index' instead.